src/app/credExplorer/WeightConfig.js

```diff
@@ -22,37 +23,45 @@ type Props = {
   onChange: (EdgeEvaluator) => void,
 };
 
+// The key should be an EdgeAddressT, but a Flow bug prevents this.
```

Comment:
Unfortunate. I have filed a bug for this:
<https://github.com/facebook/flow/issues/6569>

I don’t have any good workarounds at this time.

Once these weights become non-hard-coded, I suspect that we will be able
to properly type this map. The problem seems to only occur with literal
keys: if you have `o: {[T]: U}` then you can assign `o[t] = u`. For that
reason, I’m okay with this temporary decrease in precision.